gfw_pixetl/pixetl_prep.py

```diff
@@ -26,16 +29,27 @@ def create_geojsons(
 
     tiles: List[DummyTile] = list()
 
+    # Assume the geojson files have been created correctly already, if an
+    # even number already exist.  This is to allow progress when there are a
+    # very large number of tiles, typically zoom_14 (see GTC-3035).
+    data_lake_bucket = get_bucket()
+    target_prefix = f"{dataset}/{version}/{prefix.strip('/')}/"
+    geojsons = get_aws_files(data_lake_bucket, target_prefix, (".geojson"))
+    if len(geojsons) >= 2 and len(geojsons) % 2 == 0:
+        LOGGER.info(f"tiles.geojson and extent.geojson already exist, skipping creation of geojsons:\n{geojsons}")
+        return
+
```

Comment:
Honestly, I really don't like making a change like this. :( In order of increasing preference, I'd much rather
3. Check for the files tiles.geojson and extent.geojson explicitly, rather than assuming that if there are %2 they're the right ones. Also, I'd really rather not silently assume this as the default behavior, instead of having an option to skip creation when they're present.
2. Add an easily-removable clause and note in the resample script to skip creating the geojsons if there are a crazy number of tiles. The intent would be to remove after parallelizing in pixetl later (see next item). Though I'm still a little confused how/when you inject the manually-created tiles.geojson.
or 1. Parallelize the gathering of info for the geojsons. I understand this is the hardest and involves the most work, but it would be the cleanest, and yield lots of benefits. I suspect I could do it in an hour or two.

Reply:
OK, thanks for the comments and the further help!  I got the parallelization working, so I'm going to  close this PR and put out a separate PR for the parallelization.